src/main/groovy/com/urbancode/air/plugin/cf/helper/CFHelper.groovy

```diff
@@ -10,8 +10,13 @@ package com.urbancode.air.plugin.cf.helper
 import com.urbancode.air.CommandHelper
 import com.urbancode.air.ExitCodeException
 
+import java.nio.charset.Charset
```

Comment:
This import is never used. It's not that big of a deal, just thought I'd mention it in case you wanted to fix it before you merged.